cpp/velox/CMakeLists.txt

```diff
@@ -124,8 +124,6 @@ macro(ADD_VELOX_DEPENDENCIES)
   if(BUILD_TESTS)
     add_velox_dependency(exec::test "${VELOX_COMPONENTS_PATH}/exec/tests/utils/libvelox_exec_test_lib.a")
     add_velox_dependency(temp::path "${VELOX_COMPONENTS_PATH}/exec/tests/utils/libvelox_temp_path.a")
-  endif()
-  if(BUILD_TESTS OR BUILD_BENCHMARKS)
```

Comment:
Just verified building benchmarks doesn't depend on `libvelox_dwio_common_test_utils.a` indeed. Thanks for your fix!